src/collections/Breadcrumb/BreadcrumbSection.js

```diff
@@ -0,0 +1,95 @@
+import * as _ from 'lodash'
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import META from '../../utils/Meta'
+
+/**
+ * A section sub-component for Breadcrumb component.
+ *
+ * @see Breadcrumb
+ */
+function BreadcrumbSection(props) {
+  const {
+    active, children, className, link, href, onClick, ...rest,
+  } = props
+
+  /**
+   * Handles onClick event. If onClick is not defined, does nothing.
+   * */
+  const handleClick = (e) => {
+    if (!_.isUndefined(e)) e.preventDefault()
+    if (!_.isFunction(onClick)) return
+
+    onClick()
+  }
+
+  const classes = cx(
+    active && 'active',
```

Comment:
This is arguably cleaner than the using the util we have for this, and we may do away with this util in the future.  However, for consistency let's use the util here.  We can refactor later if we do decide to remove this simple util:

```js
import { useKeyOnly } from '../../utils/propUtils'

// ...

const classes = cx(
  useKeyOnly(active, 'active')
)
```